lib/web_ui/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ environment:
 
 dependencies:
   meta: 1.1.7
+  image: 2.1.4
```

Comment:
Does this need to be a `dev_dependencies`?

Reply:
dev_dependencies is used for files that are under the `test` directory.

Reply:
Was confusing that with "build"/"runtime" deps, thanks!